index.html

```diff
@@ -15,276 +15,291 @@
-                    accounts or sign-ups
-                    required.</p>
-                <p class="section-description1">A daily selection of privately personalized reads; no
-                    accounts or sign-ups
-                    required.</p>
-                <ul class="tags">
-                    <li class="li11">
-                        <article class="text1">html</article>
-                    </li>
-                    <li class="li222">
-                        <article class="text22"> Ruby on rails </article>
-                    </li>
-                    <li class="li22">
-                        <article class="text2">css</article>
-                    </li>
-                    <li class="li33">
-                        <article class="text3">javascript
-                    </li>
+        </div>
+
+    </section>
+    <section class="sections">
+
+    </section>
+
+    <section class="section2">
+        <div class="right-block00">
+            <img class="img-section2" src="images/Snapshoot Portfolio1.png" alt="project image">
+            <img class="img-sec2" src="images/Snapshoot Portfolio section2.png" alt="project image">
+        </div>
+        <div class="left-block00">
+            <div class="experience1">
+                <h1 class="section-title1">Multi-Post
+                    Stories</h1>
+                <ul class="list">
+                    <li class="list1">CANOPY</li>
+                    <li class="list2">Back End Dev</li>
+                    <li class="list3">2015</li>
                 </ul>
-                <div class="action">
-                    <button class="btn">See Project</button>
-                </div>
             </div>
+            <p class="section-description0">A daily selection of privately personalized reads; no
+                accounts or sign-ups
+                required.</p>
 
-        </section>
-        <section class="section4">
-            <div class="right-block00">
-                <img class="image-section4" src="images/Snapshoot Portfolio3.png" alt="project image">
-                <img class="img-sec4" src="images/Snapshoot Portfolio section 3.png" alt="project image">
+            <p class="section-description1">A smart assistant to make driving more safe, efficient, and fun by
+                unlocking
+                your most expensive computer: your car.</p>
+            <ul class="tags">
+                <li class="li11">
+                    <article class="text1">html</article>
+                </li>
+                <li class="li222">
+                    <article class="text22"> Ruby on rails </article>
+                </li>
+                <li class="li22">
+                    <article class="text2">css</article>
+                </li>
+                <li class="li33">
+                    <article class="text3">javascript
+                </li>
+            </ul>
+            <div class="action">
```

Comment:
- [ ] It would be better if we maintain the original design, here we should have 2 buttons, 1: for view project and 2: for view live version. Let us also fix the border around the text
![Screenshot from 2023-01-06 22-31-11](https://user-images.githubusercontent.com/46388113/211110675-0db6a238-f54a-48b9-b992-bae90ee2ff44.png)